helpers.js

```diff
@@ -1,33 +1,37 @@
 const ftp = require("basic-ftp");
 
-function getOptions(action, settings) {
+const DEFAULT_FTP_PORT = 21;
+
+function mapConnectOptions(action, settings) {
```

Comment:
`connect` is a verb, and you probably wanted a noun here, which would be `connection`

You're not mapping connection, but action and settings, so it's worth emphasizing that you're "mapping to connection".